### PR TITLE
owner: make unit test faster

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What problem does this PR solve?

This improves https://github.com/pingcap/tidb/issues/26624, but not completely fix it.

Problem Summary:

### What is changed and how it works?

What's Changed:

- There are two  `func TestT(t *testing.T) ` so the test case run twice.
- There is a `sleep(4s)` in the tests.
- The last problem is etcd client retry connection make test slow, but there is no workaround

How it Works:

```
make failpoint-enable
cd owner
go test -v
```

Before: 15.543s
After: 5.948s

### Check List 

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [X] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note 

```release-note
None
```

### CODE REVIEW

File: owner/manager_test.go [hunk 1]
```diff
@@ -232,7 +227,6 @@ func TestCluster(t *testing.T) {
 	if err != nil {
 		t.Fatal(err, IsNil)
 	}
-	time.Sleep(time.Duration(tmpTTL+1) * time.Second)
```
Comment (zimulala) [inline owner/manager_test.go:235]: Will the test be unstable when that sleep is removed?

Comment (lysu) [reply owner/manager_test.go:235]: +1

**Review by lysu — COMMENTED** [review]
rest LGTM

File: owner/manager_test.go (hunk 1 above)
Comment (tiancaiamao) [reply owner/manager_test.go:235]: The test is still stable after removing this sleep ...

**Review by zimulala — APPROVED** [review]
LGTM

**Review by lysu — APPROVED** [review]

**Pull request merged**